src/Components/GEOSldas_GridComp/GEOSlandpert_GridComp/land_pert.F90

```diff
@@ -503,7 +257,7 @@ subroutine LDASsa_get_pert(                         &
 
           do i=1,pert_grid_l%N_lon
 
-             call adjust_mean(pert_grid_l%N_lat, N_ens, Pert_ntrmdt(m,i,:,:) )
+             call adjust_mean(pert_grid_l%N_lat, N_ens, Pert_ntrmdt(i,:,m,:) )
```

Comment:
A silly question... We are passing a 4-dim array with two singleton dimensions into adjust_mean(), which expects a 2-dim array.  Does Fortran automatically "squeeze" the singleton dimensions and effectively present the desired 2-dim array to adjust_mean? 
In the new branch, the non-singleton dimension are no longer consecutive, as they are in the original code.

Reply:
It may be faster if if the two-D is the first two because of the continuous memory.  Fortran won't squeeze it but would remember the stride between elements.